Source/Device.generated.swift

```diff
@@ -286,6 +288,239 @@ public enum Device {
     ///
     /// ![Image](https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP808/sp808-apple-watch-series-5_2x.png)
     case appleWatchSeries5_44mm
+  #elseif os(macOS)
```

Comment:
Are you sure `os(macOS)` is true when running on a Catalyst target?

If not, we can use:
```
#elseif os(macOS) || targetEnvironment(macCatalyst)
```
to support both Catalyst and native macOS apps.

Reply:
`os(macOS)` return false on Catalyst. `os(iOS)`return true.
catalyst is an iOS device so the code compiled will be the one create before with #if os(iOS) 

I make this PR to see if catalyst must be managed well by DeviceKit, and cut my work into two features. First macOS , then [macCatalyst](https://github.com/devicekit/DeviceKit/issues/214)

Reply:
Okay that fine then.